src/Analyzer/NameResolver.php

```diff
@@ -341,4 +358,24 @@ private function resolveType($node)
 
         return $node;
     }
+
+    private function getPhpDocNode(Stmt\Property $node): ?PhpDocNode
+    {
+        if (null === $node->getDocComment()) {
+            return null;
+        }
+
+        $lexer = new Lexer();
```

Comment:
@AlessandroMinoccheri to avoid creating this objects every time we can add `Lexer` `PhpDocParser` and `TokenIterator` as class attributes: what you think?

Reply:
Yes, we can try to do it.